### PR TITLE
Refactor models

### PR DESCRIPTION
**Pull request by davidbeig**
PR with the refactoring of the models for this module.

### CODE REVIEW

File: app/views/decidim/time_tracker/time_tracker/_activity.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  assignee = Decidim::TimeTracker::Assignee.find_by(activity: activity, user: current_user)
+%>
+
+<div class="row activity" data-activity-id="<%= activity.id %>">
+  <div class="columns large-8">
+    <h2><%= translated_attribute activity.description %></h2>
+    <span>Click on the button to track your time.</span>
+    <div class="elapsed_time">Time  elapsed: <span id="elapsed_time" data-activity-id="elapsed_time_<%= activity.id %>" data-time-entry="<%= last_time_entry(assignee).to_json %>"> 0h 0m 0s </div>
+  </div>
+  <button id="start" data-endpoint="<%= Decidim::EngineRouter.main_proxy(current_component).task_activity_time_entries_path(activity.task, activity.id)%>" data-activity-id="<%= activity.id %>" class="columns large-1 button">
```
Comment (microstudi) [inline app/views/decidim/time_tracker/time_tracker/_activity.html.erb:11]: This is a little bit cumbersome, better we create a helper method in the controller.

Comment (davidbeig) [reply app/views/decidim/time_tracker/time_tracker/_activity.html.erb:11]: Ok. We'll change it with the next branch

File: app/views/decidim/time_tracker/time_tracker/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  <div id="time_tracker" class="columns">
+    <% if !user_signed_in? %>
+      <%= render partial: "sign_in" %>
+    <% end %>
+    <% tasks.each do |task| %>
+      <h2> <%= translated_attribute task.name %> </h2>
+        <div id="activities">
+          <% task.activities.each do |activity| %>
+            <% if activity.active %>
+              <% if Decidim::TimeTracker::Assignee.find_by(activity: activity, user: current_user).status == "accepted" %>
```
Comment (microstudi) [inline app/views/decidim/time_tracker/time_tracker/index.html.erb:13]: This kind of check is better to do it with a helper in the controller, something like `assignee_accepted?`

File: app/assets/config/admin/decidim_time_tracker_manifest.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+/*
+ *= link decidim/time_tracker/admin/navigation_maps.css
```
[the remark this answers is not in the document]
Comment (davidbeig) [reply app/assets/config/admin/decidim_time_tracker_manifest.css]: As you can see that file is outdated, i don't know why it appears now to you.

**Pull request merged**